plugin-gestures/src/main/java/com/mapbox/maps/plugin/gestures/GesturesPluginImpl.kt

```diff
@@ -415,37 +411,93 @@ class GesturesPluginImpl : GesturesPlugin, GesturesSettingsBase, MapStyleObserve
     centerScreen = ScreenCoordinate((width / 2).toDouble(), (height / 2).toDouble())
   }
 
+  /**
+   * Used instead of [com.mapbox.android.gestures.StandardGestureDetector.SimpleStandardOnGestureListener]
+   * since it adds NonNull annotations while its parent [android.view.GestureDetector.OnGestureListener] does not
+   * have those annotations in the release Android versions (added recently at https://android.googlesource.com/platform/frameworks/base/+/68c65e58b4f4daf79c9ffab518a826a506799db2/core/java/android/view/GestureDetector.java).
+    */
+  private open class SimpleStandardOnGestureListener : StandardOnGestureListener {
```

Comment:
could similar approach be used for other listeners from that library (if any)?

Reply:
MotionEvent is not passed to us in other detectors. 
However, in fact this issue should be fixed in the upstream Gestures [library](https://github.com/mapbox/mapbox-gestures-android/blob/6fc34d91fc8d8758c51830127c49223624d446aa/library/src/main/java/com/mapbox/android/gestures/StandardGestureDetector.java#L98). 

Created an issue in the gesture repo https://github.com/mapbox/mapbox-gestures-android/issues/100